prqlc/bindings/clib/Cargo.toml

```diff
@@ -9,9 +9,9 @@ rust-version.workspace = true
 version.workspace = true
 
 [lib]
-name = "prqlc"
 crate_type = ["staticlib", "cdylib"]
 doctest = false
+name = "prqlc_lib"
```

Comment:
Just a question: why `_lib`, not `_clib`?

Reply:
No view! 

@aljazerzen what would you prefer?

Reply:
I guess `_lib`? I don't know...